src/components/RecoverPass/index.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+//Import Material-UI
+import { Box, TextField, Button, Card, CardMedia, CardContent, Typography } from "@material-ui/core";
+//Import Formik
+import { useFormik } from "formik";
+//Import Styles
+import "./style.scss";
+//Import DATA
+import Logo from "assets/images/sushiMochi.jpg";
+
+//Import SchemaValidation
+import Schema from "./schema";
+
+const RecoverPass = () => {
+  const formik = useFormik({
```

Comment:
podrias hacer un destructuring, queda mas limpio que andar haciendo `formik.` (formik punto algo) 
```suggestion
  const { values, handleSubmit, errors } = useFormik({
```

Reply:
Done!
Quedo muchisimo mas prolijo de como estaba. Voy a tratar de acostumbrarme a hacerlo así.